### ISSUE TITLE
Implement new designs for "Legal resources" landing page

### ISSUE THREAD

**Issue by AmyKort** [18F/fec-cms#1601]
@jenniferthibault commented on [Fri Sep 01 2017](https://github.com/18F/openFEC-web-app/issues/2288)

Discussion and history in https://github.com/18F/openFEC-web-app/issues/2185 
___

### Designs
<img width="700" alt="screen shot 2017-10-25 at 10 26 59 am" src="https://user-images.githubusercontent.com/11636908/32004146-1a925960-b96f-11e7-805f-23e8bf2f9f68.png">

### Notes

**Links**
- Advanced searches
  - Advisory opinions (AOs): https://www.fec.gov/data/legal/search/advisory-opinions/?type=advisory_opinions&ao_category=F
  - Matters under review (MURs): https://www.fec.gov/data/legal/search/enforcement/?search_type=murs&search=&mur_no=&mur_respondents=
  - Audits: (link to come once search is finished)
  - Regulations: https://www.fec.gov/regulations
  - Statutes: https://www.fec.gov/data/legal/search/statutes
- All news: https://www.fec.gov/updates/
- Browse legal resources:
  - Advisory opinions: https://www.fec.gov/data/legal/advisory-opinions/
  - Enforcement of federal campaign finance law: https://www.fec.gov/legal-resources/enforcement/
  - Regulations: https://www.fec.gov/legal-resources/regulations/
  - Statutes: https://www.fec.gov/legal-resources/statutes
  - Legislation and Commission recommendations: https://www.fec.gov/legal-resources/legislation
  - Court cases: https://www.fec.gov/legal-resources/court-cases
  - Audits: https://www.fec.gov/legal-resources/enforcement/audit-reports/
  - Policy and other guidance: https://transition.fec.gov/law/policy.shtml
- Guides for candidates and committees: Link to the page created by the implementation of this issue: https://github.com/18F/openFEC-web-app/issues/2286
- Contribution limits: https://www.fec.gov/help-candidates-and-committees/candidate-taking-receipts/contribution-limits-candidates/

**Tasks to completion:**

- [x] @jenniferthibault / @JonellaCulmer finalizes and exports images, icons (should not block the rest of implementation)
- [ ] Add images & icon assets to `fec-cms`
- [x] Match to design above
- [x] Check mobile-readiness

---

@AmyKort commented on [Thu Sep 07 2017](https://github.com/18F/openFEC-web-app/issues/2288#issuecomment-327837249)

@dorothyyeager and @gsmaragdis -- just flagging this one for you.

---

@jenniferthibault commented on [Wed Oct 25 2017](https://github.com/18F/openFEC-web-app/issues/2288#issuecomment-339351990)

Image above edited to include final icon designs.

Some of these icons have already been added:
- For AO's: [the direction sign](https://github.com/18F/fec-cms/blob/release/public-20171011/fec/fec/static/icons/input/direction-sign.svg)
- For enforcement: [shield](https://github.com/18F/fec-cms/blob/release/public-20171011/fec/fec/static/icons/input/shield.svg)
- For court cases: [mallet](https://github.com/18F/fec-cms/blob/release/public-20171011/fec/fec/static/icons/input/mallet.svg) 
  - Since this icon isn't used anywhere else right now, can we change the file name to `gavel.svg` instead as part of this work?

The rest of the icons are exported as SVG's and packaged in this zip file:
📁  [legal-icons.zip](https://github.com/18F/openFEC-web-app/files/1414958/legal-icons.zip)

---

@johnnyporkchops commented on [Sun Oct 29 2017](https://github.com/18F/openFEC-web-app/issues/2288#issuecomment-340266630)

<s>@jonella, @jenniferthibault - Do you think the icon SVGs should al start at the same size for cards? We can, of course, set differing background size properties in the CSS to accommodate the different originals sizes. For example the ones on data landng inherit the card__icon background size of 50%. If I add the new ones, I have to add a specific rule for 75% for the ones that appear on legal resources landing. If these were the same size it would mean fewer lines of CSS , not big deal but thought it might be a smart coordination between design/front-end teams.</s>
![smaller_svg](https://user-images.githubusercontent.com/5572856/32144836-2060b3f4-bc95-11e7-89ae-6520c264cefa.gif)

---

@johnnyporkchops commented on [Sun Oct 29 2017](https://github.com/18F/openFEC-web-app/issues/2288#issuecomment-340267549)

Disregard the last comment regarding icons, I just checked and realized that the new icons made were the same size as most. Just a few older ones are smaller, which is not worth changing, given that they may have already been used throughout the SCSS with various bg-size  properties,
![ok_svg](https://user-images.githubusercontent.com/5572856/32144900-6fe50da2-bc96-11e7-98e6-30f10f2cd75c.gif)

---

@jenniferthibault commented on [Mon Oct 30 2017](https://github.com/18F/openFEC-web-app/issues/2288#issuecomment-340463577)

@johnnyporkchops our working standard is to build each icon at a 20x20 px square so that the line weights and spacing are consistent when scaled. You're right that a few of the earlier ones are probably not consistent to that template.

Since the gavel is being replaced overall, here it is exported as 20x20, so we can make it match.
 [gavel.svg.zip](https://github.com/18F/openFEC-web-app/files/1427550/gavel.svg.zip)

Since not all icons have perfectly square proportions, when exporting we crop the artboard to the dimensions of the art. Perhaps what you're saying is that we shouldn't do this, and instead should export all icons at 20x20, including white space outside the artwork? @xtine what do you think?

Here are some examples from the working file, where you can see that the checklist, CFR and U.S.C. icons aren't full 20x20:
![screen shot 2017-10-30 at 10 34 39 am](https://user-images.githubusercontent.com/11636908/32176414-f7621730-bd5d-11e7-8370-1d6b7b28f9f8.png)

---

@johnnyporkchops commented on [Wed Nov 01 2017](https://github.com/18F/openFEC-web-app/issues/2288#issuecomment-341177169)

The non-square seem to  work fine in the card icons even though they have a different height, I was more concerned with the older, smaller sizes. I will replace mallet with gavel at the new size. Thanks!

---

@xtine commented on [Wed Nov 01 2017](https://github.com/18F/openFEC-web-app/issues/2288#issuecomment-341180734)

@jenniferthibault: personally I'm not concerned with all icons being 20x20 because each icon needs a specific class defined for them, so the positioning is defined in the stylesheet.

---

@patphongs commented on [Mon Nov 13 2017](https://github.com/18F/openFEC-web-app/issues/2288#issuecomment-344007127)

Had a question for @jenniferthibault regarding the card design on the legal resources landing page: https://github.com/18F/fec-cms/pull/1443.

**Comment by jenniferthibault** [comment]
@AmyKort since the work for this was merged into develop via https://github.com/18F/fec-cms/pull/1443, would you like to close it?

**Comment by AmyKort** [comment]
Yep. Thanks!
